packages/documentation/copy/en/handbook-v2/Object Types.md

```diff
@@ -579,7 +579,7 @@ draw({ color: "blue", radius: 42 });
 draw({ color: "red", raidus: 42 });
 ```
 
-## Interfaces vs. Intersections
+## Interface Extension vs Intersection
```

Comment:
```suggestion
## Interface Extension vs. Intersection
```

Sorry, one last nit; didn't notice the `.` removal. This has to be written `vs.` or `Versus`

Reply:
That and, this will change the anchor links to this section; so hopefully nobody's linked to this page before...

Reply:
I personally like it more without the dot, but I will accept your suggestion nonetheless.